operator/pkg/central/values/translation/translation.go

```diff
@@ -195,15 +205,15 @@ func getCentralPersistenceValues(p *platform.Persistence, checker *pvcStateCheck
 	return &persistence, nil
 }
 
-func getCentralComponentValues(c *platform.CentralComponentSpec, checker *pvcStateChecker, obsoletePVC bool) (*translation.ValuesBuilder, error) {
+func getCentralComponentValues(c *platform.CentralComponentSpec, m *platform.GlobalMonitoring, checker *pvcStateChecker, obsoletePVC bool) (*translation.ValuesBuilder, error) {
 	cv := translation.NewValuesBuilder()
 
 	cv.AddChild(translation.ResourcesKey, translation.GetResources(c.Resources))
 	if c.DefaultTLSSecret != nil {
 		cv.SetMap("defaultTLS", map[string]interface{}{"reference": c.DefaultTLSSecret.Name})
 	}
 
-	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled())
+	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled() || m.IsOpenShiftMonitoringEnabled())
```

Comment:
What does this line do exactly? Note that we want to be able to configure `central.exposeMonitoring` (http endpoint, default `false`) and `monitoring.openshift.enabled` (https endpoint, default `true`) separately.

Reply:
So this line exposes monitoring if it's enabled or if openshift monitoring that's uses exposed monitoring endpoint. I think we may add a todo here to change it once monitoring endpoint will be available.